acis_thermal_check/main.py

```diff
@@ -35,6 +36,9 @@
           "less_equal": "<="}
 
 
+use_noon_day_start()
```

Comment:
This needs a comment, like: Without using this then the epoch times in the existing model spec files are interpreted using midnight for `YYYY:DDD` dates while the regression data are for `YYYY:DDD:12:00:00`. This compatibility shim makes everything work without changing all the configured model spec files.